src/sentry/types/region.py

```diff
@@ -192,7 +192,13 @@ def get_region_by_name(name: str) -> Region:
     try:
         return global_regions.by_name[name]
     except KeyError as e:
-        raise RegionResolutionError(f"No region with name: {name!r}") from e
+        region_names = [
+            r.name for r in global_regions.regions if r.category == RegionCategory.MULTI_TENANT
+        ]
+        raise RegionResolutionError(
+            f"No region with name: {name!r} "
+            f"(expected one of {region_names!r} or a single-tenant name)"
+        ) from e
```

Comment:
This might not be a long-term problem, but it's come up often enough recently that I wanted an easier way to tell what was in my local config.

I figure it's safe enough to print the list of all multi-tenant regions, but single-tenant regions could grow to unbounded numbers and might also leak customer identities.